### ISSUE TITLE
[FEATURE] Horizontal tooltips

### ISSUE THREAD

**Issue by Moldoteck** [bokeh/bokeh#11455]
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
When hovering over multiple glyphs, there will be displayed multiple tooltips in vertical manner. Would be nice if we would be able to choose their alignment(vertical or horizontal). 

Even more, in the future, an option could be added to display them in a grid, like 3x3, but for now horizontal alignment should be enough

**Comment by bryevdv** [comment]
cc @bokeh/dev I am include to close this with `noaction` The default basic tooltip is intended to be that, a very basic default. For anything else, there is the option of custom HTML tooltips:

https://docs.bokeh.org/en/latest/docs/user_guide/tools.html#custom-tooltip

**Comment by mattpap** [comment]
Even with custom tooltips it's non-trivial to control the appearance of collections of tooltips, so to me this is definitively a feature worth considering. Especially given the upcoming CSS layout improvements should allow implementing this fairly easily.

**Comment by Moldoteck** [comment]
> Even with custom tooltips it's non-trivial to control the appearance of collections of tooltips, so to me this is definitively a feature worth considering. Especially given the upcoming CSS layout improvements should allow implementing this fairly easily.

Agree, I have worked with html tooltips - quite nice, but I haven't managed to control collections of tooltips
